### PR TITLE
Make segments clearer, add to customers

### PR DESCRIPTION
**Pull request by demoore**

### CODE REVIEW

File: v1/segments.md [hunk 1]
```diff
@@ -39,7 +39,19 @@
 {
     "email_id": "tem_ACdWZKZf4CtZNPM27WAdf6",
     "email_data": {
-        "name": "John Doe"
+        "Weekly_Newsletter":
+        [
+            {
+                "url": "http://www.example.com/1",
+                "image": http://www.example.com/image1.jpg",
```
Comment (brandonb927) [inline v1/segments.md]: Missing a beginning dub-quote here

File: v1/segments.md [hunk 2]
```diff
@@ -39,7 +39,19 @@
...
+                "text": "Check this sweet thing out!"
+            },
+            {
+                "url": "http://www.example.com/2",
+                "image": http://www.example.com/image2.png",
```
Comment (brandonb927) [inline v1/segments.md]: also missing a beginning dub-quote here

Comment (demoore) [inline v1/segments.md]: gor blimey

**Comment by brandonb927** [comment]
:+1:

**Pull request merged**